### PR TITLE
Material design

### PR DESCRIPTION
**Pull request by ryanto**
Here is my first pass at adding something like material design.

Some of the UI elements aren't exactly to spec, but everything is working/functioning fine. Also, I touched up a few components in the process of doing this.

Home page

App view

### CODE REVIEW

**Comment by samselikoff** [comment]
woah...dude !

this looks insanely awesome!

**Comment by samselikoff** [comment]
trying to run it and getting this weird error

```
Cannot convert a Symbol value to a string
TypeError: Cannot convert a Symbol value to a string
    at RegExp.test (native)
    at Symbol.hasOwnProperty (/Users/SamSelikoff/TED/front_end_builds/admin/node_modules/ember-cli/node_modules/broccoli-es6modules/node_modules/es6-module-transpiler/dist/es6-module-transpiler.js:163:43)
    at /Users/SamSelikoff/TED/front_end_builds/admin/node_modules/ember-cli-6to5/node_modules/broccoli-6to5-transpiler/node_modules/6to5-core/node_modules/core-js/library.js:136:17
    at setToStringTag (/Users/SamSelikoff/TED/front_end_builds/admin/node_modules/ember-cli-6to5/node_modules/broccoli-6to5-transpiler/node_modules/6to5-core/node_modules/core-js/library.js:88:13)
    at /Users/SamSelikoff/TED/front_end_builds/admin/node_modules/ember-cli-6to5/node_modules/broccoli-6to5-transpiler/node_modules/6to5-core/node_modules/core-js/library.js:610:3
    at /Users/SamSelikoff/TED/front_end_builds/admin/node_modules/ember-cli-6to5/node_modules/broccoli-6to5-transpiler/node_modules/6to5-core/node_modules/core-js/library.js:614:2
    at Object.<anonymous> (/Users/SamSelikoff/TED/front_end_builds/admin/node_modules/ember-cli-6to5/node_modules/broccoli-6to5-transpiler/node_modules/6to5-core/node_modules/core-js/library.js:2101:2)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
```

**Comment by samselikoff** [comment]
btw did you see this? https://github.com/miguelcobain/ember-paper

File: admin/app/mirage/data/apps.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 export default [
   {
     id: 1,
-    name: 'blog',
+    name: 'Blog',
```
Comment (samselikoff) [inline admin/app/mirage/data/apps.js]: is this representative of what will come over from the backend? i believe this is from the rails router code, i.e.

```rb
front_end :blog
```

**Comment by hhff** [comment]
This bootiful

Was just about to post ember-paper!

:heart:

File: admin/app/mirage/data/apps.js (hunk 1 above)
Comment (ryanto) [inline admin/app/mirage/data/apps.js]: Yah. Will change back.

**Comment by ryanto** [comment]
> Cannot convert a Symbol value to a string

I had that, can't remember what I did to fix. I think blowing away node_modules.

**Pull request merged**